pygeoapi/provider/xarray_.py

```diff
@@ -92,6 +92,8 @@ def __init__(self, provider_def):
             self.axes = [self._coverage_properties['x_axis_label'],
                          self._coverage_properties['y_axis_label'],
                          self._coverage_properties['time_axis_label']]
+            self.time_axis_covjson = provider_def.get('time_axis_covjson') \
+                or self.time_field
```

Comment:
What is the point of adding this? Can we add a test case that uses this field. Suggest to potentially rename in pygeoapi, but will defer to @tomkralidis. This addition should also be noted in the docs ([OACoverages](https://github.com/geopython/pygeoapi/blob/master/docs/source/data-publishing/ogcapi-coverages.rst) and [OAEDR](https://github.com/geopython/pygeoapi/blob/master/docs/source/data-publishing/ogcapi-edr.rst)